pkg/harvester-manager/list/harvesterhci.io.management.cluster.vue

```diff
@@ -79,19 +94,78 @@ export default {
       extensionsLink: {
         name:   'c-cluster-uiplugins',
         params: { cluster: BLANK_CLUSTER }
-      },
+      }
     };
   },
 
+  watch: {
+    async harvesterRepository(value) {
```

Comment:
I don't think there's an action point here, but it's looks a bit weird to me this watch. I've been thinking about this and an `async` computed is not a good pattern, so I guess this is the next best thing, am I right?

Reply:
The logic could be moved from this watch hook to the `fetch()`, but in that case the page would wait sync to calculate the latest Harvester version before rendering.
Another approach is to use promise.then() but the await is cleaner.